### ISSUE TITLE
Hardcoded api_timeout value makes updating firmware impossible.

### ISSUE THREAD

**Issue by cnf** [dell/dellemc-openmanage-ansible-modules#454]
### Bug Description

Trying to use redfish_firmware in ansible, I get a `<urlopen error The write operation timed out>` error.

Changing the api_timeout to 300 makes it work

https://github.com/dell/dellemc-openmanage-ansible-modules/blob/collections/plugins/modules/redfish_firmware.py#L179-L180

### Component or Module Name

redfish_firmware 

### Ansible Version

Ansible 2.13.5

### Python Version

Python 3.8.12

### iDRAC/OME/OME-M version

IDRAC 5.10.50

### Operating System

Not Applicable 

### Playbook Used

```yaml
- name: Update Firmware
   dellemc.openmanage.redfish_firmware:
     baseuri: {{ idrac_ip }}
     username: {{ idrac_username }}
     password: {{idrac_password }}
     image_url: “firmware/iDRAC-with-Lifecycle-Controller_Firmware_RTTPH_WN64_5.10.50.00_A00.EXE”
     timeout: 300 
```

### Logs
```
TASK [Update Firmware] ***************************
fatal: [worker-11]: FAILED! => {“changed”: false, “msg”: “<urlopen error The write operation timed out>”}
```
### Steps to Reproduce

Have a connection / vpn that is slow enough the firmware can’t get uploaded within the hard-coded timeout window.

### Expected Behavior

For the `timeout:` parameter, that is documented, to actually be applied instead of the hardcoded value

### Actual Behavior

The `api_timeout` is hard-coded to 100 seconds, which is not enough in my current case

**Comment by anupamaloke** [comment]
@cnf, thank you reporting this bug. I have added it to the backlog and we will fix this as part of 7.1.0 release in Dec.

**Comment by sachin-apa** [comment]
Closing the issues as this is fixed with the Release 7.1.0 https://github.com/dell/dellemc-openmanage-ansible-modules/pull/465
